registry.json

```diff
@@ -978,6 +978,20 @@
     "operatorName": "",
     "github": "CO2-Pool"
   },
+  {
+    "ticker": "SOIL",
+    "poolId": "48b18f581c5e77a19870e8399e4d2100c66b2e15243c9557e79f2165",
+    "social": {
+      "twitter": "Ada4Soil",
+      "website": "https://ada4.io/soil",
+      "facebook": "",
+      "instagram": " ",
+      "youtube": " "
+    },
+    "telegram": "ADA4 SOIL",
+    "operatorName": "ADA4",
```

Comment:
Please can you change operatorName to either a your real name or leave it blank?